### DISCUSSION TITLE
Getting CSS selector in "Previous patterns" to work

### DISCUSSION THREAD

**Discussion by unaimed** [gdh1995/vimium-c#843] [Q&A] [answered]
Hello all,

I am trying to use a CSS selector as a pattern for the "goPrevious"-command.

The following issue in the Vimium project suggested to me that something like this would be possible: [#3739](https://github.com/philc/vimium/issues/3739)

I cannot seem to get it to work and I am struggling to find a reference to this feature in the documentation.

The element I am trying to find:

`<a class="Navigation Navigation-prev" history="[object Object]" location="[object Object]" href="/gallery/8NR1Qve"><div class="content"><svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" alt="<"><title>Chevron Pointing Left</title><path d="M10 12L6 8.004 10 4" stroke="#ffffff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path></svg></div></a>`

My attempt at the "Previous patterns":
`prev,previous,back,older,<,‹,←,«,≪,<<,"a.Navigation.Navigation-prev"`

Can somebody point me in the right direction?

**Comment by unaimed** [comment]
After trying some more this solution worked for me:

prev,previous,back,older,<,‹,←,«,≪,<<,**[class="Navigation Navigation-prev"]**

A big thanks for this great extension!

**Comment by gdh1995** [comment, marked as answer]
you may use .Navxxxxx, if only you remove the quote characters in your first comment:


****,a.Nav.Nav-prev,*****



---Original---
From: "Kevin ***@***.***&gt;
Date: Sun, Jan 15, 2023 21:45 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [gdh1995/vimium-c] Getting CSS selector in "Previous patterns" towork (Discussion #843)




 
After trying some more this solution worked for me:
 
prev,previous,back,older,<,‹,←,«,≪,<<,[class="Navigation Navigation-prev"]
 
A big thanks for this great extension!
